### PR TITLE
Cancel STT on going to App Drawer

### PR DESCRIPTION
**Pull request by zfir**

### CODE REVIEW

File: src/android/com/pbakondy/SpeechRecognition.java [hunk 1]
```diff
@@ -76,6 +77,12 @@ public void run() {
     });
   }
 
+  @Override
+  public void onStop() {
+    super.onStop();
+    callbackContext.error(BACKGROUND_ERROR);
```
Comment (azharbeebeejaun) [inline src/android/com/pbakondy/SpeechRecognition.java:83]: error only if it's running ? Sinon on va avoir une erreur à chaque fois qu'on quitte l'appli ? (Pas sur)

Comment (zfir) [reply src/android/com/pbakondy/SpeechRecognition.java:83]: @azharbeebeejaun Yes if there is an instantiated SpeechRecogniser class, it will then trigger.

Comment (zfir) [reply src/android/com/pbakondy/SpeechRecognition.java:83]: Helpful: https://github.com/distinctdan/cordova-plugin-extra-events

Comment (zfir) [reply src/android/com/pbakondy/SpeechRecognition.java:83]: Removed Use [cordova-plugin-extra-events](https://github.com/distinctdan/cordova-plugin-extra-events) instead

**Pull request closed without merging**